blocks/content-api-source-block/package.json

```diff
@@ -2,13 +2,8 @@
   "name": "@wpmedia/content-api-source-block",
   "version": "5.14.1",
   "description": "Content source block for Content API queries",
-  "author": "Joe Grosspietsch <joe.grosspietsch@washpost.com>",
```

Comment:
you don't like beltran and joe?

Reply:
Nothing against them.  Just so many hands in this that an author is not really worth including, IMO.  Especially if they leave and we have to go update/clean/remove.